src/Pim/Component/Catalog/Repository/ProductRepositoryInterface.php

```diff
@@ -29,13 +29,6 @@
     public function buildByChannelAndCompleteness(ChannelInterface $channel);
 
     /**
-     * @param array $ids
-     *
-     * @return \Doctrine\Common\Collections\ArrayCollection
-     */
-    public function findByIds(array $ids);
```

Comment:
yep, but only on the old implementation of cursors (which will be removed after the pre review)